config/config.go

```diff
@@ -211,8 +212,9 @@ func validateReferencesExist(c *configpb.Configuration) error {
 // TestGrid file prefix, post-normalization.
 func validateName(s string) error {
 	name := normalize(s)
-	if name == "" {
-		return errors.New("normalized name can't be empty")
+
+	if len(name) < 3 {
+		return errors.New("names must contain at least 3 alphanumeric characters")
```

Comment:
Shouldn't there also be a maximum length?

Reply:
Maybe. What should it be?

Reply:
I'd just pick a longish number and ensure it doesn't exceed this (such as 512, 1024 characters). Doesn't particularly matter.